Makefile

```diff
@@ -41,10 +48,34 @@ test-integration:
 	@# Help: Run integration tests
 	mvn integration-test -Dskip.unit.tests=true
 
+,PHONY: run-local
```

Comment:
Should this not be .PHONY instead than ,PHONY, please? This is the same also in the charges-delta-consumer, just noticed now...

Reply:
@gseregni-ch I was confused about this as well as I saw same change in charges-delta-consumer and in insolvency-delta-consumer